aiken.html.markdown

```diff
@@ -0,0 +1,134 @@
+---
+language: Aiken
+contributors:
+    - ["Riley-Kilgore", "http://github.com/Riley-Kilgore"]
+filename: learnaiken.ak
+---
+
+Aiken is a programming language designed specifically for writing smart contracts on the Cardano blockchain. It compiles to Untyped Plutus Lambda Calculus (UPLC), which evaluates on the Cardano-Node.
+
+The language includes features tailored for blockchain operations and smart contract development. These include built-in functions based on what's available in UPLC, with upcoming additions like BLS12-381 curve primitives.
+
+Aiken comes with a standard library and prelude that are particularly useful for creating Cardano smart contracts. The language supports various programming constructs such as functions, validators, pattern matching, and algebraic data types.
+
+Aiken accomodates to developer experience with innovations like CIP-57 (Blueprints) having been implemented. This allows developers to write less code to build transactions that are compliant with their validators.
+
+As a relatively new language, Aiken is evolving to meet the needs of the Cardano ecosystem. Its development is guided by the requirements of smart contract creation on the Cardano Blockchain.
+
+```aiken
+//// Basic Syntax and Structure
+
+// Comments in Aiken start with '//'
+// Module-level comments use '////'
+// Documentation comments use '///'
+//// This is a module-level comment
+/// This is a documentation comment
+
+// Constants are defined using 'const'
+const my_constant: Int = 42
+
+// Functions are defined using 'fn'
+fn add(a: Int, b: Int) -> Int {
+  a + b
+}
+
+// Public functions are marked with 'pub'
+pub fn public_function(x: Int) -> Int {
+  x * 2
+}
+
```

Comment:
It would be nice to also add illustrations for:

- Function capturing
- Pipeline
- Backpassing